### PR TITLE
mail-filter/mailfilter: bump to 0.8.7

### PR DESCRIPTION
**Pull request by hhoffstaette**
This bump fixes several bugs resolved since 0.8.4.

Bug: https://bugs.gentoo.org/574164
Bug: https://bugs.gentoo.org/587134
Bug: https://bugs.gentoo.org/769407

Signed-off-by: Holger Hoffstätte <holger@applied-asynchrony.com>

### CODE REVIEW

File: mail-filter/mailfilter/mailfilter-0.8.7.ebuild [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+SRC_URI="mirror://sourceforge/mailfilter/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~sparc ~x86"
+IUSE="+ssl"
+
+DEPEND="sys-devel/flex
+	ssl? ( dev-libs/openssl:* )"
+RDEPEND=""
```
Comment (thesamesam) [inline mail-filter/mailfilter/mailfilter-0.8.7.ebuild:17]: This seems wrong:
1. RDEPEND too, surely
2. :* isn't likely to be correct if it links against it. Does it? (This affects 1.). Will need := instead.

File: mail-filter/mailfilter/mailfilter-0.8.7.ebuild [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (thesamesam) [inline mail-filter/mailfilter/mailfilter-0.8.7.ebuild:4]: 8?

File: mail-filter/mailfilter/mailfilter-0.8.7.ebuild (hunk 1 above)
Comment (hhoffstaette) [reply mail-filter/mailfilter/mailfilter-0.8.7.ebuild]: Yeah..not sure what happened here. I've changed to :0= and added openssl to RDEPEND.

File: mail-filter/mailfilter/mailfilter-0.8.7.ebuild (hunk 2 above)
Comment (hhoffstaette) [reply mail-filter/mailfilter/mailfilter-0.8.7.ebuild]: Fixed.

**Pull request closed without merging**